tools/record.sh

```diff
@@ -41,7 +47,7 @@ invalid_input() {
 
 check_deps
 
-while getopts ":ph" opt
+while getopts "d:pht" opt
```

Comment:
```suggestion
while getopts ":phtd:" opt
```

This should be ":phtd:", ":" at the start of a getopt argument prevents it from displaying debug errors.